### DISCUSSION TITLE
Scotte Chip 4.06

### DISCUSSION THREAD

**Discussion by lindberg87** [motoz/PellMon#106] [Q&A]
Hey Pellmon.

Which version of pellmon will you recommend me to install on my raspberry pi then I have a Scotte 4.06 chip ?
I had tried the v0.7.0 version with a usb FTDI but its cant connect to the scotte 4.06 chip. The raspberry pi fine sees the ftdi device.

Best Regards Lindberg

**Comment by motoz** [comment]
It is unfortunately entirely possible that your chip is too old to work. I only know that the oldest version did some things differently (like checksums in the transmission data), but I have no info on exactly what and how. I know for a fact that at least some of the newer 4.x chips works with PellMon, but if you can't connect then it is possible that your version doesn't.

**Comment by lindberg87** [comment]
Thank you for answer my question.
I have found out that the rs232 communication chip are missing on the controller board. So there for it can't communicate with the ftdi seriel device. I will buy a chip and that should do it.

Best regards Stig

Mvh Stig Lindberg


________________________________
Fra: Anders Nylund <notifications@github.com>
Sendt: Saturday, December 19, 2020 11:00:03 AM
Til: motoz/PellMon <PellMon@noreply.github.com>
Cc: lindberg87 <st1z@live.dk>; Author <author@noreply.github.com>
Emne: Re: [motoz/PellMon] Scotte Chip 4.06 (#106)


It is unfortunately entirely possible that your chip is too old to work. I only know that the oldest version did some things differently (like checksums in the transmission data), but I have no info on exactly what and how. I know for a fact that at least some of the newer 4.x chips works with PellMon, but if you can't connect then it is possible that your version doesn't.
